responsibleai_text/responsibleai_text/utils/genai_metrics/metrics.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+"""Compute AI-assisted metrics for generative text models."""
+
+from pathlib import Path
+
+import evaluate
```

Comment:
can you wrap this in a try/catch similar to this one:
```
try:
    import evaluate
except ImportError:
    module_logger.debug(
        'Could not import evaluate, required if using a QA model')
```

Reply:
https://github.com/microsoft/responsible-ai-toolbox/blob/main/responsibleai_text/responsibleai_text/rai_text_insights/rai_text_insights.py#L38

Reply:
otherwise it can cause errors for other tasks which don't need it, specifically if __init__ files re-import functions for shorter namespaces